### ISSUE TITLE
Middle click to open file

### ISSUE THREAD

**Issue by AshlessCinder** [atom/github#900]
It would be cool if middle clicking (the scroll wheel) on a file in the staged/unstaged changes window opened that file. 

It's a super common action for me, and I would love the quick middle click option instead of having to right click and select "open file"

**Comment by ron-wolf** [comment]
Definitely. A double-click resulting in staging added a bit to the learning curve for me.

**Comment by marekmurawski** [comment]
Middle-click is a bit debatable for opening files, but definitely some way to quickly open a file would be awesome. Consider `alt + click`?

**Comment by AshlessCinder** [comment]
In browsers, middle click opens a link in a new tab. This is very similar to that.
